ironfish-cli/src/commands/deposit.ts

```diff
@@ -201,7 +201,21 @@ Find the transaction on https://explorer.ironfish.network/transaction/${
         errorReason: `No graffiti found. Register at ${REGISTER_URL} then run \`ironfish testnet\` to configure your graffiti`,
       }
     }
-    const user = await this.api.findUser({ graffiti })
+
+    let user
+    try {
+      user = await this.api.findUser({ graffiti })
+    } catch (error: unknown) {
+      if (error instanceof Error) {
+        this.error(`API error: ${error.message}`)
```

Comment:
We can omit `API Error`

```suggestion
        this.error(error.message)
```